app/auth/Main.hs

```diff
@@ -32,6 +32,7 @@ withEcho :: Bool -> IO a -> IO a
 withEcho echo action = do
   putStr "Enter password: " >> hFlush stdout
   old <- hGetEcho stdin
+  putStr "\n" >> hFlush stdout
```

Comment:
I don't quite understand this.  I'm used to password prompts like this:

![image](https://user-images.githubusercontent.com/1779/71014608-0ee90180-20a7-11ea-9493-ca3072b25957.png)

The existing code does the same thing:

![image](https://user-images.githubusercontent.com/1779/71014687-317b1a80-20a7-11ea-9277-3cf08a6efd5e.png)

Do you have things that behave differently?

Reply:
Yes.  after taking the input, it just put the prompt there.  Needs a newline to put the prompt on the next line.
![image](https://user-images.githubusercontent.com/81302/71550346-24054080-2983-11ea-8085-df7489302ac2.png)